src/components/home/Home.tsx

```diff
@@ -1,12 +1,89 @@
-import './Home.css';
+import "./Home.css";
+import teas from "../../teas.png";
+import { NavLink } from "react-router-dom";
 
 function Home() {
-
-    return (
-        <>
-        
-        </>
-    )
+  return (
+    <div>
+      <div className="intro-wrapper">
+        <article className="article-intro">
+          <img src={teas} />
+          <div className="info">
+            <h2 className="home-h2">Home</h2>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+              aliquip ex ea commodo consequat.
+            </p>
+            <button className="infoButton">
+              <NavLink to="/tea/black">See Black tea's →</NavLink>
```

Comment:
Update the button text for each tea be updated to omit the apostrophe

Reply:
On it!